gcloud/orb.yaml

```diff
@@ -445,6 +460,14 @@ jobs:
           Name of your deployment. If you are using multi-deployment
           autoscaling this may correspond to multiple actual k8s deployments.
         type: string
+      executor:
+        default: gcloud-alpine-latest
+        description: >
+          Executor for the job. Must have a working gcloud version installed.
+          To use your own custom executor, see
+          `https://circleci.com/docs/2.0/reusing-config/#authoring-reusable-executors`.
+          Defaults to a small google/cloud-sdk:alpine docker image.
+        type: executor
```

Comment:
Perhaps I'm rusty with how thee are used as my intuition says the line should look like `executor: <<parameters.executor>>`.